src/pages/team/components/holder/holder.jsx

```diff
@@ -1,66 +1,98 @@
 import React from "react";
-import { Box, Paper } from "@material-ui/core";
-import IconButton from "@material-ui/core/IconButton";
-import styles from "./style.module.css";
-import Female from "../../../../assets/team/_person.svg";
-import Male from "../../../../assets/team/person.svg";
+import { Grid, Box, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from 'prop-types';
+
 import GitHub from "../../../../assets/team/github_.svg";
 import Linkedin from "../../../../assets/team/linkedin.svg";
 import Instagram from "../../../../assets/team/instagram.svg";
 import Medium from "../../../../assets/team/medium.svg";
 
+const useStyles = makeStyles({
+    container: {
+        margin: '9px',
+        boxShadow: '7px 7px #F8B406',
+    },
+    gridContainer: {
+        // display: 'flex',
+        // flexDirection: 'column',
+        // justifyContent: 'center',
+    },
+    memberImage: {
+        height: '100px',
+        width: '100px',
+    },
+    links: {
```

Comment:
You don't need to use this, you can just pass props in the material-ui component and it will act accordingly to the breakpoints